docs/testplan/Smartswitch-test-plan.md

```diff
@@ -0,0 +1,874 @@
+# Test plan for Smartswitch
```

Comment:
I am seeing it mostly only includes the test case for platform / PMON. Is this intentional?

If so, do you mind to rename the PR and file to SmartSwitch PMON or Platform test plan or something like this? The SmartSwitch also contains many other things, such as VNET scenarios, HA, reboot and etc, so the title of the doc and PR can be very misleading.

Reply:
@nissampa agree with Riff. This looks platform test plan

Reply:
Changed it to smartswitch platform as the title